dags/resources/stages/parse/table_definitions/abracadabra/BentoBoxMarkets_event_LogRepayForAll.json

```diff
@@ -0,0 +1,54 @@
+{
+    "parser": {
+        "abi": {
+            "anonymous": false,
+            "inputs": [
+                {
+                    "indexed": false,
+                    "internalType": "uint256",
+                    "name": "amount",
+                    "type": "uint256"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint128",
+                    "name": "previousElastic",
+                    "type": "uint128"
+                },
+                {
+                    "indexed": false,
+                    "internalType": "uint128",
+                    "name": "newElastic",
+                    "type": "uint128"
+                }
+            ],
+            "name": "LogRepayForAll",
+            "type": "event"
+        },
+        "contract_address": "SELECT cloneAddress FROM ref('BentoBoxV1_event_LogDeploy')",
```

Comment:
@AyadiGithub I'm not super familiar with this, but to me it looks like the [`deploy` function of `BentoBoxV1`](https://etherscan.io/address/0xf5bce5077908a1b7370b9ae04adc565ebd643966#code#L435) allows cloning an arbitrary contract. I checked a few `cloneAddress` values and have not encountered one that implemented `LogRepayForAll` (e.g. [1](https://etherscan.io/address/0x8cff769b0ab77c98cfc06d71046c2d81cfbf43ee#code), [2](https://etherscan.io/address/0x1c5771e96C9d5524fb6e606f5B356d08C40Eb194#code), [3](https://etherscan.io/address/0xc8f2e98f4b3a8a428742cb2f23ac644120a5ec3d#code)). Conversely, do we know all `LogRepayForAll` events emitted by contracts deployed this way can always be attributed to abracadabra? Should there not at least be a reference to a specific `masterContract` in this query?

Reply:
@TimNooren LogRepayForAll is a new function and event for Abracadabra's new V4 Cauldrons. I do have a list of masterContracts used by Abracadabra; however, I think adding them to the query would not future-proof the table definition (due to the possibility of new masterContracts for future Cauldrons). I can add them if you think its better!

Reply:
I guess we can try it like this👍